src/components/node/person_profile/index.tsx

```diff
@@ -0,0 +1,118 @@
+                width={meta?.width}
+                height={meta?.height}
+              />
+            </div>
+          )}
+          <div className="vads-u-display--flex vads-u-flex-direction--column">
+            <h1 className="vads-u-font-size--xl vads-u-margin-bottom--0p5">
+              {title}
+              {suffix && [suffix]}
+            </h1>
+            {fieldDescription && (
+              <p className="vads-u-font-size--lg vads-u-margin-bottom--0p5">
+                {fieldDescription}
+              </p>
+            )}
+            {fieldEmailAddress && (
+              <p className="vads-u-font-size--lg vads-u-margin-bottom--0p5">
+                <span className="vads-u-font-weight--bold">Email: </span>
+                <a target="blank" href="mailto:{{ fieldEmailAddress }}">
+                  {fieldEmailAddress}
+                </a>
+              </p>
+            )}
+            {fieldPhoneNumber && (
+              <p
+                className="
+                vads-u-font-weight--regular
+                vads-u-margin--0
+                vads-u-margin-bottom--0p5"
+              >
+                <span className="vads-u-font-weight--bold">Phone: </span>
+                <a href="tel:{{ fieldPhoneNumber }}">{fieldPhoneNumber}</a>
+              </p>
+            )}
+          </div>
+        </div>
+        {fieldCompleteBiographyCreate && (
+          <div className="va-introtext vads-u-margin-bottom--2">
+            <p className="va-introtext vads-u-margin-bottom--0">
+              {fieldIntroText}
+            </p>
+            <div
+              className="vads-u-margin-bottom--2"
+              dangerouslySetInnerHTML={{ __html: fieldBody.processed }}
+            />
+          </div>
+        )}
+        {fieldPhotoAllowHiresDownload && (
+          <div
+            className="vads-u-align-content--flex-end va-c-margin-top--auto vads-u-margin-bottom--2"
+            id="download-full-size-photo-link"
+          >
+            <i
+              className="va-c-social-icon fas fa-download"
+              aria-hidden="true"
+            ></i>
+            <a href={image} download>
+              Download full size photo
+            </a>
+          </div>
+        )}
+        {fieldCompleteBiography && (
+          <div
+            className="vads-u-align-content--flex-end va-c-margin-top--auto vads-u-margin-bottom--2"
+            id="download-full-size-photo-link"
+          >
+            <i
+              className="va-c-social-icon fas fa-download"
+              aria-hidden="true"
+            ></i>
+            <div className="vads-u-align-content--flex-end va-c-margin-top--auto vads-u-margin-bottom--2">
+              <i className="va-c-social-icon fas fa-download"></i>
+              <a href={fieldCompleteBiography.url} download>
+                Download full bio (PDF)
+              </a>
+            </div>
+          </div>
+        )}
+      </article>
+      <hr />
+    </div>
+  )
+}
```

Comment:
Person profile component with conditionals to match [person_profile](https://github.com/department-of-veterans-affairs/content-build/blob/master/src/site/layouts/person_profile.drupal.liquid) layout on content-build